src/ClusterTSnePlot.js

```diff
@@ -125,7 +147,9 @@ ClusterTSnePlot.propTypes = {
 
   loading: PropTypes.bool.isRequired,
   resourcesUrl: PropTypes.string,
-  errorMessage: PropTypes.string
+  errorMessage: PropTypes.string,
+
+  tooltipContent: PropTypes.func
```

Comment:
I’m not really sure if `tooltipContent` should be a prop. I think you could export `fetchResponseJson` in `TSnePlotView`, import it here, and you’d have the function in the same file where it’s used.

Reply:
Good, as long as you weighed down the options. I don’t think the endpoint is likely to change, but we have more important stuff to care about!